.travis.yml

```diff
@@ -1,23 +1,27 @@
 # see http://about.travis-ci.org/docs/user/languages/php/ for more hints
 language: php
+
 php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
```

Comment:
@Soullivaneuh: Your example does not show if a build will finish (is stopped) or continue as soon as a job has failed, which was the question.

Let's consider this `fast_finish` option as a separate matter from this particular issue.